app/cli/cmd/root.go

```diff
@@ -83,6 +93,38 @@ func NewRootCmd(l zerolog.Logger) *cobra.Command {
 			return nil
 		},
 		PersistentPostRunE: func(cmd *cobra.Command, args []string) error {
+			// Do not track telemetry if the env variable is set
+			var telemetryClient telemetry.Client
+			if os.Getenv(doNotTrackEnv) == "1" || os.Getenv(doNotTrackEnv) == "true" {
+				logger.Debug().Msg("Telemetry is disabled")
+			} else {
+				var err error
+				telemetryClient, err = posthog.NewClient(posthogAPIKey, posthogEndpoint)
```

Comment:
so you do not want to crash on purpose correct?

Reply:
Yes, I don't want to crash if the telemetry fails, I think that could disturb users and automations